plugin/documents.py

```diff
@@ -159,3 +160,10 @@ def on_query_context(self, key: str, operator: int, operand: Any, match_all: boo
 
     def __str__(self) -> str:
         return str(self.view.id())
+
+    def on_hover(self, point: int, hover_zone: int) -> None:
```

Comment:
Would try to move it to a better position in the file :)

Reply:
Is 36d1af3 better?